build/relsignDSCModules.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/bash
+
+# Script to sign DSC modules and drop off signed packages (in zip format) into build share.
+# It generates a checksum file for each DSC module, uploads to azure storage account using SAS_SOURCE_URI,
+# then downloads the signed *.asc file from SAS_DEST_URI and 
+# serializes it into the zip package for given DSC module.
```

Comment:
we (oms ) are calling it from jenkins, but the script is written in such a way it can be called from anywhere since it  does not have any GitHub project hierarchical dependencies whatsoever. I will add more details in the description if it would help.